biopet/biopet.wdl

```diff
@@ -24,15 +24,16 @@ import "../common.wdl"
 
```

Comment:
Missing some parameters meta, but we are going to remove this anyways right?

Reply:
Indeed, the biopet folder will be removed. I have not added parameter_meta to tasks that have not been updated to the SLURM standard. So if the tasks doesn't have time_minutes and in some cases DockerImage.